independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/core/strategy/ContentMergeCodestartFileStrategyHandler.java

```diff
@@ -40,6 +40,10 @@ public void process(Path targetDirectory, String relativePath, List<TargetFile>
             return;
         }
         createDirectories(targetPath);
-        writeFile(targetPath, template.get().getContent().replace("{merged-content}", mergedContent.toString()));
```

Comment:
so basically only write it if it has no contents?

I get that it solves it for now ...but even empty files are worth having.

no other way we can handle it ?

Reply:
Added @deprecated